src/pybind/mgr/status/module.py

```diff
@@ -260,9 +260,9 @@ def handle_fs_status(self, cmd):
                 output += version_table.get_string() + "\n"
 
         if output_format == "json":
-            return 0, "", json.dumps(json_output, sort_keys=True)
+            return 0, json.dumps(json_output, sort_keys=True), ""
```

Comment:
instead of returning a tuple, could return an instance of `HandleCommandResult`.

Reply:
Done